workos/directory_sync.py

```diff
@@ -131,6 +138,7 @@ def list_directories(
             "limit": limit,
             "before": before,
             "after": after,
+            "organization_id": organization,
```

Comment:
Same thing here. I think we can leave this up by `domain`, since it fits there better since they're both filters.

Reply:
👍🏽